src/ontology/cl-edit.owl

```diff
@@ -32727,6 +32729,26 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:7107724") oboInOwl:hasRe
 AnnotationAssertion(rdfs:label obo:CL_4052011 "follicle associated enterocyte")
 SubClassOf(obo:CL_4052011 obo:CL_0000584)
 
+# Class: obo:CL_4052015 (endocrine gland capillary endothelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18480313") obo:IAO_0000115 obo:CL_4052015 "Any capillary endothelial cell that is part of the endocrine gland.")
+AnnotationAssertion(terms:contributor obo:CL_4052015 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052015 "2024-09-30T15:27:49Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4052015 "endocrine gland capillary endothelial cell")
+SubClassOf(obo:CL_4052015 obo:CL_0002144)
+SubClassOf(obo:CL_4052015 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002368))
+
+# Class: obo:CL_4052016 (pituitary gland capillary endothelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10810312") Annotation(oboInOwl:hasDbXref "PMID:32910242") Annotation(oboInOwl:hasDbXref "PMID:4899902") obo:IAO_0000115 obo:CL_4052016 "A capillary endothelial cell that is part of the pituitary gland, particularly in the anterior pituitary. This cell is distinguished by its numerous fenestrations, which are covered by a single-layered diaphragm. The diaphragm features a dense ring-like structure, facilitating the passage of hormones and other substances essential for endocrine signalling.")
+AnnotationAssertion(terms:contributor obo:CL_4052016 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052016 "2024-09-30T15:31:45Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:4899902") oboInOwl:hasNarrowSynonym obo:CL_4052016 "capillary endothelial cell of the anterior pituitary")
+AnnotationAssertion(rdfs:label obo:CL_4052016 "pituitary gland capillary endothelial cell")
+EquivalentClasses(obo:CL_4052016 ObjectIntersectionOf(obo:CL_0002144 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000007)))
+SubClassOf(obo:CL_4052016 obo:CL_4052015)
+SubClassOf(obo:CL_4052016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000007))
```

Comment:
> [pituitary gland capillary endothelial cell](http://purl.obolibrary.org/obo/CL_4052016) EquivalentTo [capillary endothelial cell](http://purl.obolibrary.org/obo/CL_0002144) and ([part of](http://purl.obolibrary.org/obo/BFO_0000050) some [pituitary gland](http://purl.obolibrary.org/obo/UBERON_0000007))
> 
> [pituitary gland capillary endothelial cell](http://purl.obolibrary.org/obo/CL_4052016) SubClassOf [part of](http://purl.obolibrary.org/obo/BFO_0000050) some [pituitary gland](http://purl.obolibrary.org/obo/UBERON_0000007)
> 
This is redundant with EC axiom.  Should be deleted.

> [pituitary gland capillary endothelial cell](http://purl.obolibrary.org/obo/CL_4052016) SubClassOf [endocrine gland capillary endothelial cell](http://purl.obolibrary.org/obo/CL_4052015)

Should be inferred from logical def of parent class.